src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -852,6 +857,45 @@ ApiGlobe.prototype.removeEventListenerLayerChanged = function removeEventListene
     this.viewerDiv.removeEventListener('layerchanged:index', this.callbackLayerChanged, false);
 };
 
+/**
+ * Add a control.
+ * @constructor
+ * @param {object} Control - The Control object.
+ */
+
+ApiGlobe.prototype.addControl = function addControl(control) {
+    var index = this.controls.indexOf(control);
+    if (index > -1) {
+        this.controls.splice(index, 1);
+    }
+    this.controls.push(control);
+    control.setGlobe(this.scene.map);
+};
+
+/**
+ * Returns all controls.
+ * @constructor
+ * @return     {array}  The array of controls.
+ */
+
+ApiGlobe.prototype.getControls = function getControls() {
+    return this.controls;
+};
+
+/**
+ * Remove a control.
+ * @constructor
+ * @param {object} Control - The Control object.
+ */
+
+ApiGlobe.prototype.removeControl = function removeControl(control) {
+    var index = this.controls.indexOf(control);
+    if (index > -1) {
+        this.controls.splice(index, 1);
+    }
+    control.setGlobe();
```

Comment:
Let's be explicit `control.setGlobe(null);`

But it sounds like an abuse to me. I would prefer having a `control.dispose` method that would remove listeners and stuff.